src/main/webapp/style.css

```diff
@@ -8,10 +8,32 @@ hidden {
 }
 
 .container {
-  margin-top: 0px;
+  margin-top: 10px;
   text-align: center;
 }
-
+ul {
+    list-style-type: none;
+}
 .selector-for-some-widget {
   box-sizing: content-box;
 }
+
+#comments {
```

Comment:
Checkout Bootstrap margins and spacing utilities so you don't have to write your own margin styles: https://getbootstrap.com/docs/4.0/utilities/spacing/